src/index.tsx

```diff
@@ -28,15 +25,23 @@ const giveBirth :any = throttle(function (mouse,numberOfBirths,colors,changeDire
   }
 
 
-} , 200)
+} , 50)
 const setFromEvent = (e,setMousePosition,boundary) =>{
   setMousePosition({ x: e.clientX - boundary.current.x, y: e.clientY - boundary.current.y });
 }
-const ReactFirefly = (props)=>{
-  let {numberOfBirths, canvasWidth, canvasHeight,colors,changeDirectionFrequency,randomMotion} = props
-  if(numberOfBirths === null){
-    numberOfBirths = 2
-  }
+const ReactFirefly = (props:props)=>{
```

Comment:
Better to use
```
interface ReactFireFlyPropTypes {
 canvasWidth:number ;
 canvasHeight:number;
 colors:string[]
}
export const DataGridDemo: React.FC<ReactFireFlyPropTypes> = ({
    canvasWidth, 
    canvasHeight, 
    colors 
 }) => { 
    .......
    return (<Canvas canvasRef={canvas} ></Canvas>)
} 
```
much more readable and clean!!! @endurance21

Reply:
loved that idea ! you can make changes and comit to this pr?

Reply:
Yup!! sure i will make all the changes and make necessary updates in doc. by the EOD.